Apps/RaptureIntTests/src/test/java/rapture/httpapi/blob/BlobApiTests.java

```diff
@@ -68,21 +64,10 @@ public void beforeTest(@Optional("http://localhost:8665/rapture")String url, @Op
           dataProvider = "blobFileScenarios", enabled=true)
     public void testBlobFromFile(String fileName, String contentType) throws IOException {  
 
-        String authorityName = "test.blob" + Thread.currentThread().getId();
-        String blobRepoUri=RaptureURI.builder(BLOB, authorityName).build().toString();
-        Reporter.log("Created "+blobRepoUri, true);
-        if(!blobApi.blobRepoExists(blobRepoUri)){
-            String blobConfig = String.format(repoConfigTemplate, "MONGODB", authorityName);
-            String metaConfig = String.format(metaConfigTemplate, "MONGODB", authorityName);
-            Reporter.log("Creating "+blobRepoUri + " with config "+metaConfig,true);
-            try {
-                blobApi.createBlobRepo(blobRepoUri, blobConfig,metaConfig);
-            }
-            catch (Exception e) {
-                Assert.fail();
-            }
-        }
-        String blobUri = blobRepoUri + "testblob" + System.nanoTime();
+    	RaptureURI repo = helper.getRandomAuthority(Scheme.BLOB);
+        helper.configureTestRepo(repo, "MONGO");
```

Comment:
Shouldn't it be MONGODB?

Hmm, I looked at the code. Some places accept either MONGO or MONGODB; others require MONGODB. We should fix that.